subclass-sandbox/src/main/java/com/iluwatar/subclasssandbox/App.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * The MIT License
+ * Copyright (c) 2014-2016 Ilkka Seppälä
+ * <p>
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * <p>
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * <p>
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.iluwatar.subclasssandbox;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The subclass sandbox pattern describes a basic idea, while not having a lot
+ * of detailed mechanics. You will need the pattern when you have several similar
+ * subclasses. If you have to make a tiny change, then change the base class,
+ * while all subclasses shouldn't have to be touched. So the base class has to be
+ * able to provide all of the operations a derived class needs to perform.
+ */
+public class App {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(App.class);
+
+  /**
+   * Entry point of the main program.
+   * @param args Program runtime arguments.
+   */
+  public static void main(String[] args) {
+    LOGGER.info("Use superpower: sky launch");
+    Superpower skyLaunch = new SkyLaunch();
```

Comment:
You can use Java's local variable type inference (var) in code like this. Please check it elsewhere too.

Reply:
Fixed in a627f17a72df44cf99d83a41fb261c93911d20cd

@iluwatar  Thanks for your suggestion. Since I do not have much experience on using var, could you kindly review my change again to see if it is ok? Thanks.